NewForce-Cohort-9/bad-blood-brotherhood
# Rev branch3

## What Changed
1. Added HTML for Liechtenstein page
2. Completed CSS for aforementioned html page
3. Added all database entries for Liechtenstein


## How to Test – ***NAME YOUR BRANCH!!!***
1.	Switch to my branch in terminal rev-branch3
2.     Check proper database format and valid page layout
